hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlTestSupport.java

```diff
@@ -432,6 +433,35 @@ public static void createMapping(HazelcastInstance instance, String name, String
         }
     }
 
+    /**
+     * Create an IMap index with given {@code name}, {@code type} and {@code attributes}.
+     */
+    public static void createIndex(String name, String mapName, IndexType type, String... attributes) {
+        createIndex(instance(), name, mapName, type, attributes);
+    }
+
+    static void createIndex(HazelcastInstance instance, String name, String mapName, IndexType type, String... attributes) {
+        SqlService sqlService = instance.getSql();
+
+        StringBuilder sb = new StringBuilder("CREATE INDEX IF NOT EXISTS ");
+        sb.append(name);
+        sb.append(" ON ");
+        sb.append(mapName);
+        sb.append(" ( ");
+        for (int i = 0; i < attributes.length; ++i) {
+            if (attributes.length - i - 1 == 0) {
+                sb.append(attributes[i]);
+            } else {
+                sb.append(attributes[i]).append(", ");
+            }
+        }
+        sb.append(" ) ");
+        sb.append(" TYPE ");
+        sb.append(type.name());
+
+        sqlService.execute(sb.toString());
```

Comment:
Isn't this much simpler?
```suggestion
        instance.getMap(mapName)
                .addIndex(new IndexConfig(type, attributes).setName(name));
```
Also, IMO, this is too simple and also too specific to be worth adding to the widely-reused class, I'd keep it only in the test that uses it.

Reply:
That was the point -- I want to have not only programmatical but also SQL-driven index creation mechanism :)